dms_instance.module

```diff
@@ -51,13 +50,17 @@ function dms_instance_dms_instance_insert(EntityInterface $entity) {
 }
 
 function dms_instance_insert_update(EntityInterface $entity) {
+  $module_handler = \Drupal::service('module_handler');
+  $module_path = $module_handler->getModule('dms_instance')->getPath();
+  require_once($module_path . DIRECTORY_SEPARATOR . 'dms_instance_secrets.inc');
   $instanceStatus = $entity->get('instance_status')->value;
   if ($instanceStatus == 16) {
     $uuid = $entity->uuid();
+    /** @var Drupal\dms_instance\Queue\DMSInstanceQueue; $queue */
     $queue = new DMSInstanceQueue($entity->getAegirInstance(), Database::getConnection());
     $queue->createQueue();
     $queue->createItem([
-      'url' => $entity->getName() . '-dms.canadahelps.org',
+      'url' => $entity->getName() . '-dms.' . SITE_DOMAIN_HOST,
```

Comment:
Where is this defined?

Reply:
its in the dms_instance_secrets.inc file which is excluded from being stored in this repository